README.md

```diff
@@ -1 +1,12 @@
 # goal-service
+
+```text
```

Comment:
No need to write TODOs here .
Rather add a section how to compile and Run the app

Reply:
This is just for me to check the todo list , onces its done I will remove these.